iothub/service/src/Jobs/ScheduledJobsClient.cs

```diff
@@ -0,0 +1,382 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.Devices.Common.Exceptions;
+using Microsoft.Azure.Devices.Http2;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Scheduled jobs management.
+    /// </summary>
+    /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-jobs"/>.
+    public class ScheduledJobsClient
+    {
+        private readonly string _hostName;
+        private readonly IotHubConnectionProperties _credentialProvider;
+        private readonly HttpClient _httpClient;
+        private readonly HttpRequestMessageFactory _httpRequestMessageFactory;
+
+        private const string JobsUriFormat = "/jobs/v2/{0}";
+        private const string JobsQueryFormat = "/jobs/v2/query";
+        private const string CancelJobUriFormat = "/jobs/v2/{0}/cancel";
+
+        private const string ContinuationTokenHeader = "x-ms-continuation";
+        private const string PageSizeHeader = "x-ms-max-item-count";
+
+        private static readonly TimeSpan s_defaultOperationTimeout = TimeSpan.FromSeconds(100);
+
+        /// <summary>
+        /// Creates client, provided for unit testing purposes only.
+        /// </summary>
+        protected ScheduledJobsClient()
+        {
+        }
+
+        internal ScheduledJobsClient(string hostName, IotHubConnectionProperties credentialProvider, HttpClient httpClient, HttpRequestMessageFactory httpRequestMessageFactory)
+        {
+            _hostName = hostName;
+            _credentialProvider = credentialProvider;
+            _httpClient = httpClient;
+            _httpRequestMessageFactory = httpRequestMessageFactory;
+        }
+
+        /// <summary>
+        /// Explicitly open the client instance.
+        /// </summary>
+        public virtual Task OpenAsync()
+        {
+            return TaskHelpers.CompletedTask;
+        }
+
+        /// <summary>
+        /// Closes the client instance and disposes its resources.
+        /// </summary>
+        public virtual Task CloseAsync()
+        {
+            return TaskHelpers.CompletedTask;
+        }
+
+        /// <summary>
+        /// Gets the job with the specified Id.
+        /// </summary>
+        /// <param name="jobId">Id of the job to retrieve</param>
```

Comment:
punctuation and simpler verbs.

```suggestion
        /// <param name="jobId">Id of the job to get.</param>
```